app/views/health/release_forms/_form.haml

```diff
@@ -44,4 +44,17 @@
 
     .col-6
       .form__actions--right
+        %button.btn.btn-secondary(aria-label="Cancel" pjax-modal-close type="button") Cancel
         = f.button :submit, 'Save', class: 'btn btn-primary'
+
+:javascript
+  $('#form_participation_signature_on').on('change', function(event) {
+    if ($('#form_signature_on').val() == '') {
```

Comment:
where is `#form_signature_on`?  Would it make more sense to do this in the controller?

Reply:
These are the two date pickers at the top of the dialog, and this fills in the other one if one is filled in, and the other is blank. We could do this in the controller, but this seemed simpler than active cable or similar.